### ISSUE TITLE
FCM installed but not working in Jupyter notebook

### ISSUE THREAD

**Issue by KNAGSAGAR** [omadson/fuzzy-c-means#74]
![FCM_Issue](https://github.com/omadson/fuzzy-c-means/assets/134188824/6fd55998-0d30-4449-82e4-e64fc3cb62e5)

FuzzyC-Means is installed in the Jupyter notebook via pip install but it wont work please see the attachment.

thanks

**Comment by omadson** [comment]
Hello @KNAGSAGAR. Can you provide more information about this question?

You can use general GitHub etiquette.

 - Write in complete sentences and avoid writing run-on sentences. Grammatical mistakes for non-native speakers of English are of course understandable, so long as an honest effort made.
 - More importantly, when requesting help, it is expected that users provide example code, example outputs, and a description of expected output. When relevant, it can be helpful to specify system information and pertinent package versions.

**Comment by KNAGSAGAR** [comment]
Hello @omadson Thanks for your response!
I have a requirement to use Fuzzy C-Means clustering for an academic mini-project using Python. In order to use Fuzzy C-Means I first installed it in my Jupyter notebook using pip install -- it got installed without any issue. Then for using the function  I imported the function using "from fcmeans import FCM " command, that's where I encountered the issue, when I run this command Iam getting "TypeError: dataclass_transform() got an unexpected keyword argument 'field_specifiers'" error. I have attached a snip for reference. This is at the library import level so Iam not able to proceed any further.

Python version on my computer is "Python 3.9.12 (main, Apr  4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)]"
Jupyter notebook version is 6.4.8
I installed it using Anaconda navigator with version 2.4.1

Thanks!!

![FCM_Import_Error](https://github.com/omadson/fuzzy-c-means/assets/134188824/f994f24a-1ce1-4e09-945a-b2308e37aa7a)



> Hello @KNAGSAGAR. Can you provide more information about this question?
> 
> You can use general GitHub etiquette.
> 
> * Write in complete sentences and avoid writing run-on sentences. Grammatical mistakes for non-native speakers of English are of course understandable, so long as an honest effort made.
> * More importantly, when requesting help, it is expected that users provide example code, example outputs, and a description of expected output. When relevant, it can be helpful to specify system information and pertinent package versions.

Hello @omadson Thanks for your response!
I have a requirement to use Fuzzy C-Means clustering for an academic mini-project using Python. In order to use Fuzzy C-Means I first installed it in my Jupyter notebook using pip install -- it got installed without any issue. Then for using the function  I imported the function using "from fcmeans import FCM " command, that's where I encountered the issue, when I run this command Iam getting "TypeError: dataclass_transform() got an unexpected keyword argument 'field_specifiers'" error. I have attached a snip for reference. This is at the library import level so Iam not able to proceed any further.

Python version on my computer is "Python 3.9.12 (main, Apr  4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)]"
Jupyter notebook version is 6.4.8
I installed it using Anaconda navigator with version 2.4.1

Thanks!!
![FCM_Import_Error](https://github.com/omadson/fuzzy-c-means/assets/134188824/fe675388-fd15-43d7-89ed-77295fdf1d47)
